.gitignore

```diff
@@ -206,4 +206,8 @@ src/Core/Properties/launchSettings.json
 *.override.env
 **/*.DS_Store
 src/Admin/wwwroot/lib
-src/Admin/wwwroot/css
\ No newline at end of file
+src/Admin/wwwroot/css
+.vscode/launch.json
```

Comment:
Should be able to use `**/*.vscode` to ignore all .vscode, e.g.
```
.vscode
**/*.vscode
```

Reply:
Hmmm, since .vscode is a folder, wouldn't it need to look like this instead?
.vscode/*
**/.vscode/*

Reply:
yes, my bad.

Reply:
Fixed